### PR TITLE
internal clk gating

### PR DESCRIPTION
**Pull request by davideschiavone**
this is a TRL3 fix for #2

### CODE REVIEW

File: cve2.core [hunk 1]
```diff
@@ -2,18 +2,15 @@ CAPI=2:
 # Copyright lowRISC contributors.
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
-name: "lowrisc:ibex:ibex_core:0.1"
-description: "Ibex CPU Core Components"
+
```
Comment (christian-herber-nxp) [inline cve2.core:5]: this is not related to #2 and thus shouldn't be a part of this PR

File: cve2.core [hunk 2]
```diff
@@ -2,18 +2,15 @@ CAPI=2:
...
+description: "OpenHW Group RISC-V Core CVE2"
 
 filesets:
   files_rtl:
     depend:
       - lowrisc:prim:assert
-      - lowrisc:prim:clock_gating
-      - lowrisc:prim:lfsr
       - lowrisc:ibex:ibex_pkg
-      - lowrisc:ibex:ibex_icache
```
Comment (christian-herber-nxp) [inline cve2.core:15]: i guess also not #2 related

File: rtl/cve2_sleep_unit.sv [hunk 3]
```diff
@@ -0,0 +1,118 @@
+// Copyright 2020 Silicon Labs, Inc.
```
Comment (christian-herber-nxp) [inline rtl/cve2_sleep_unit.sv:1]: same comments on license and copyrights

File: rtl/ibex_core.sv [hunk 4]
```diff
@@ -350,13 +350,42 @@ module ibex_core import ibex_pkg::*; #(
   // for RVFI
   logic        illegal_insn_id, unused_illegal_insn_id; // ID stage sees an illegal instruction
 
-  //////////////////////
-  // Clock management //
-  //////////////////////
+  //////////////////////////////////////////////////////////////////////////////////////////////
```
Comment (christian-herber-nxp) [inline rtl/ibex_core.sv:353]: is this some sort of consistent style in open hw group?

File: bhv/cve2_sim_clock_gate.sv [hunk 5]
```diff
@@ -0,0 +1,30 @@
+// Copyright 2017 ETH Zurich and University of Bologna.
```
[the remark this answers is not in the document]
Comment (davideschiavone) [reply bhv/cve2_sim_clock_gate.sv:1]: I do not think we should change HEADERS from files to our own licence but keep the original

File: cve2.core [hunk 6]
```diff
@@ -2,18 +2,15 @@ CAPI=2:
 # Copyright lowRISC contributors.
```
[the remark this answers is not in the document]
Comment (davideschiavone) [reply cve2.core:2]: I do not agree, only new files should have the new header
